react-frontend/src/components/Nav/navbar.js

```diff
@@ -131,6 +133,15 @@ function NavBar() {
                 Products & Services
               </NavBarLink>
             </NavBarLi>
+            <NavBarLi>
+              <NavBarLinkExternal href="https://digital.gov.bc.ca/marketplace">
+                Marketplace
+                <FontAwesomeIcon
+                  icon={faExternalLinkAlt}
+                  style={{ paddingLeft: '5px', height: 'inherit' }}
```

Comment:
Yes, I think the external link icon and opening in a new tab/window is good.

However, this is what I see when I run the branch:

![Screen Shot 2020-11-23 at 11 07 03 AM](https://user-images.githubusercontent.com/16946297/100004553-84ac2180-2d7c-11eb-88e7-238c28d26e71.png)


When I inspect the icon (svg), the `height: inherit` seems to be causing it. Maybe just remove the height property here?

Reply:
hmm, which browser?

Reply:
should be fixed now

Reply:
It was on Safari. All fixed now.